src/rpc/mempool.cpp

```diff
@@ -888,34 +890,57 @@ static RPCHelpMan submitpackage()
             Chainstate& chainstate = EnsureChainman(node).ActiveChainstate();
             const auto package_result = WITH_LOCK(::cs_main, return ProcessNewPackage(chainstate, mempool, txns, /*test_accept=*/ false));
 
-            // First catch any errors.
+            std::map<Wtxid, std::optional<std::string>> wtixd_to_error_strings;
```

Comment:
wtxid
```suggestion
            std::map<Wtxid, std::optional<std::string>> wtxid_to_error_strings;
```

Reply:
also, why have this intermediate map? can't you do this in the loop at the end that populates `rpc_result`?

Reply:
removed the cruft